{"name": "test_multiple_login_page_success[invalid_user4-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user4'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753512711503, "stop": 1753512712581, "uuid": "a0c8b6b6-1e00-4cb4-879d-8ae110ce713c", "historyId": "8169f900955b8e47d48e3638b02b295f", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "68007-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}